relay.go

```diff
@@ -552,6 +552,15 @@ func (r *Relayer) handleNonCallReq(f *Frame) error {
 		return nil
 	}
 
+	if f.messageType() == messageTypeCallRes {
+		cr, err := newLazyCallRes(f)
+		if err != nil {
+			r.logger.Error(err.Error())
```

Comment:
This won't have much useful context if we log errors in this way, can we have more information:
 * a message indicating that a call res frame was not parsed
 * context on the call (e.g., the ID, any info about the item that may be useful)